app/src/interfaces/_system/system-display-template/system-display-template.vue

```diff
@@ -76,20 +78,33 @@ const injectValue = computed(() => {
 
 	return { fields: [fakeVersionField] };
 });
+
+const tree = computed(() => {
+	if (props.fields) {
+		return { list: props.fields };
+	}
+
+	if (collection.value === null) {
+		return null;
+	}
+
+	const { treeList, loadFieldRelations } = useFieldTree(collection, injectValue);
+
+	return { list: treeList.value, pathLoader: loadFieldRelations };
+});
 </script>
 
 <template>
 	<div class="system-display-template">
-		<v-notice v-if="collection === null">
-			{{ t('interfaces.system-display-template.select_a_collection') }}
+		<v-notice v-if="tree === null">
+			{{ t('interfaces.system-display-template.select_a_collection_or_tree_fields') }}
 		</v-notice>
-
 		<v-field-template
 			v-else
-			:collection="collection"
+			:tree="tree.list"
 			:model-value="value"
 			:disabled="disabled"
-			:inject="injectValue"
+			:load-path-level="tree.pathLoader"
 			@update:model-value="$emit('input', $event)"
 		/>
```

Comment:
Would be neat if we could include the `placeholder` pro of `v-field-template` so that it looks more consistent with the other fields together. `v-field-template` does have the ability to show placeholders:

![image](https://github.com/directus/directus/assets/14810858/b0c35087-a31a-4183-8f94-1d8f78d4aa22)

The others already show the example URL. We'd need to add it to the yaml under options and pass it down.

Reply:
I believe I left it out until the question regarding the default value was addressed. Agreed that it makes sense either way.

Reply:
Added via https://github.com/directus/directus/pull/21174/commits/41d308046b3cfb97802bafeb85e7305e104a9228